File: src/led/json/MetaInfo.java
package led.json;

import com.google.gson.annotations.SerializedName;

public class MetaInfo {
	@SerializedName("led_count")
	public int mLedCount;
	
	@SerializedName("repeat_count")
	public int mRepeatCount;
	
	@SerializedName("led_formats")
	public String[] mLEDFormats;
	
	@SerializedName("led_format")
	public String mLEDFormat;
	
	@SerializedName("file_version")
	public String mFileVersion;
}

File: src/led/json/MetaInfo.kt
package led.json

import com.google.gson.annotations.SerializedName

class MetaInfo {
    @SerializedName("led_count")
    var mLedCount = 0

    @SerializedName("repeat_count")
    var mRepeatCount = 0

    @SerializedName("led_formats")
    lateinit var mLEDFormats: Array<String>

    @SerializedName("led_format")
    var mLEDFormat: String? = null

    @SerializedName("file_version")
    var mFileVersion: String? = null
}